.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -121,3 +121,22 @@ body:
       placeholder: v1.24.1
     validations:
       required: true
+
+  - type: input
+    id: postgresql_version
+    attributes:
+      label: PostgreSQL version
+      description: What version of PostgreSQL are you running? (see Database information dashboard -> PostgreSQL Version (upper right on desktop) or database -> image section in your `docker-compose.yml`)
+      placeholder: v17
+
+  - type: checkboxes
+    attributes:
+      label: Are you running latest PostgreSQL version?
+      description: |
+        Please ensure you are running the latest major supported PostgreSQL version by comparing your installation with the (installation docs)[https://docs.teslamate.org/docs/installation/docker/].
+
+        If not, please first do a backup and follow [Upgrading PostgreSQL to a new major version](https://docs.teslamate.org/docs/maintenance/upgrading_postgres)
```

Comment:
backup is actually part of the guide but better mention it twice :)

Reply:
Yeah, I know but people keep going over without doing point 1 the backup :-)